cli/ripsaw/util/exceptions.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+
+class BenchmarkFailedError(Exception):
+    def __init__(self, name, uuid, msg=None):
+        if msg is None:
+            msg = f"The benchmark {name} with uuid {uuid} failed"
+        super(BenchmarkFailedError, self).__init__(msg)  # pylint: disable=super-with-arguments
```

Comment:
Is there a reason why this was disabled rather than changing the line to the following?

```python
super().__init__(msg)
```

Reply:
fixed